### DISCUSSION TITLE
Output Environment Url in Set Connection Variables

### DISCUSSION THREAD

**Discussion by YannickRe** [microsoft/powerplatform-build-tools#517] [Ideas]
Using Set Connection Variables, it already outputs ClientId, ClientSecret, TenantId and ConnectionString (which includes the Environment URL). Would it be possible to output that Environment Url as a separate output variable for us to use in subsequent tasks?

It's essentially a 4 line code change, but I'm not sure if there is a reason why it's not in there currently 😅

**Comment by petrochuk** [comment]
It is open-source project :) feel free to send Pull Request with your proposal.

**Reply by YannickRe** [reply]
I wanted to make sure there isn't a security reason why you aren't already doing it, but given this comment I created the PR: https://github.com/microsoft/powerplatform-build-tools/pull/518

Turns out it was only one line 🙂

**Reply by jboliveira** [reply]
@YannickRe / @petrochuk hands-up; We don't have the PR merged yet.

**Reply by YannickRe** [reply]
Anything you expect from me as contributor to do?

**Comment by YannickRe** [comment]
This has been merged, let's close the discussion.
